include/swift/Basic/SourceLoc.h

```diff
@@ -239,16 +303,15 @@ class CharSourceRange {
     return contains(Other.getStart()) || Other.contains(getStart());
   }
 
-  StringRef str() const {
-    return StringRef(Start.Value.getPointer(), ByteLength);
-  }
+  StringRef str() const { return StringRef(Start.Pointer, ByteLength); }
```

Comment:
What an evil method 😬 All the usages should use `SourceManager::extractText()` instead. (Not in this PR)

Reply:
😅